plugin/evm/service.go

```diff
@@ -92,198 +79,6 @@ func (service *AvaxAPI) Version(r *http.Request, _ *struct{}, reply *VersionRepl
-func (service *AvaxAPI) ImportKey(r *http.Request, args *client.ImportKeyArgs, reply *api.JSONAddress) error {
-	log.Info("EVM: ImportKey called", "username", args.Username)
-
-	if args.PrivateKey == nil {
-		return errMissingPrivateKey
-	}
-
-	reply.Address = atomic.GetEthAddress(args.PrivateKey).Hex()
-
-	service.vm.ctx.Lock.Lock()
-	defer service.vm.ctx.Lock.Unlock()
-
-	db, err := service.vm.ctx.Keystore.GetDatabase(args.Username, args.Password)
-	if err != nil {
-		return fmt.Errorf("problem retrieving data: %w", err)
-	}
-	defer db.Close()
-
-	user := user{db: db}
-	if err := user.putAddress(args.PrivateKey); err != nil {
-		return fmt.Errorf("problem saving key %w", err)
-	}
-	return nil
-}
-
-// ImportAVAX is a deprecated name for Import.
-func (service *AvaxAPI) ImportAVAX(_ *http.Request, args *client.ImportArgs, response *api.JSONTxID) error {
-	return service.Import(nil, args, response)
-}
-
-// Import issues a transaction to import AVAX from the X-chain. The AVAX
-// must have already been exported from the X-Chain.
-func (service *AvaxAPI) Import(_ *http.Request, args *client.ImportArgs, response *api.JSONTxID) error {
-	log.Info("EVM: ImportAVAX called")
-
-	chainID, err := service.vm.ctx.BCLookup.Lookup(args.SourceChain)
-	if err != nil {
-		return fmt.Errorf("problem parsing chainID %q: %w", args.SourceChain, err)
-	}
-
-	service.vm.ctx.Lock.Lock()
-	defer service.vm.ctx.Lock.Unlock()
-
-	// Get the user's info
-	db, err := service.vm.ctx.Keystore.GetDatabase(args.Username, args.Password)
-	if err != nil {
-		return fmt.Errorf("couldn't get user '%s': %w", args.Username, err)
-	}
-	defer db.Close()
-
-	user := user{db: db}
-	privKeys, err := user.getKeys()
-	if err != nil { // Get keys
-		return fmt.Errorf("couldn't get keys controlled by the user: %w", err)
-	}
-
-	var baseFee *big.Int
-	if args.BaseFee == nil {
-		// Get the base fee to use
-		baseFee, err = service.vm.estimateBaseFee(context.Background())
-		if err != nil {
-			return err
-		}
-	} else {
-		baseFee = args.BaseFee.ToInt()
-	}
-
-	tx, err := service.vm.newImportTx(chainID, args.To, baseFee, privKeys)
-	if err != nil {
-		return err
-	}
-
-	response.TxID = tx.ID()
-	if err := service.vm.mempool.AddLocalTx(tx); err != nil {
-		return err
-	}
-	service.vm.atomicTxPushGossiper.Add(&atomic.GossipAtomicTx{Tx: tx})
-	return nil
-}
-
-// ExportAVAX exports AVAX from the C-Chain to the X-Chain
-// It must be imported on the X-Chain to complete the transfer
-func (service *AvaxAPI) ExportAVAX(_ *http.Request, args *client.ExportAVAXArgs, response *api.JSONTxID) error {
-	return service.Export(nil, &client.ExportArgs{
-		ExportAVAXArgs: *args,
-		AssetID:        service.vm.ctx.AVAXAssetID.String(),
-	}, response)
-}
-
-// Export exports an asset from the C-Chain to the X-Chain
-// It must be imported on the X-Chain to complete the transfer
-func (service *AvaxAPI) Export(_ *http.Request, args *client.ExportArgs, response *api.JSONTxID) error {
```

Comment:
This is not recommended for mainnet, but if we think this is still in use we can restore this
https://docs.avax.network/api-reference/c-chain/api#avaxexport

Reply:
Same applies for other APIs as well.